core-developers/developer-log.rst

```diff
@@ -1,3 +1,4 @@
+.. _developer-log:
 .. _developers:
```

Comment:
Why two targets?

Reply:
Keeps the old target working (for `#fragments`) and ensures the filename is always a target as a simple rule-of-thumb.

A